pipeline/gke-deploy-build-push/0.1/gke-deploy-build-push.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: gke-deploy-build-push
+  labels:
+    app.kubernetes.io/version: '0.1'
+  annotations:
+    tekton.dev/pipelines.minVersion: '0.12.1'
+    tekton.dev/tags: build, push, deploy
+spec:
+  description: >-
+    This Pipeline builds, pushes, and deploys your application to a Google Kubernetes
+    Engine cluster using gke-deploy.
+  workspaces:
+    - name: source
```

Comment:
can we add a bit description about workspace as well so that folks using CLI could get to know why and for what purpose this workspace used?